modules/custom/az_eds/az_eds_user/config/install/az_eds_user.az_user_role_query.az_eds_user_administrators.yml

```
@@ -0,0 +1,7 @@
+langcode: en
+status: true
+dependencies: {  }
+id: az_eds_user_administrators
+label: 'Quickstart Administrators'
+query: az_eds_user_cws
+role: administrator
```

The lack of new line here will get flagged by PHPCS if we merge #4990.
```suggestion
role: administrator

```